consensus/core/src/synchronizer.rs

```diff
@@ -1025,7 +1026,6 @@ impl<C: NetworkClient, V: BlockVerifier, D: CoreThreadDispatcher> Synchronizer<C
                     let peer_hostname = &context.committee.authority(peer_index).hostname;
                     match response {
                         Ok(fetched_blocks) => {
-                            info!("Fetched {} blocks from peer {}", fetched_blocks.len(), peer_hostname);
```

Comment:
why remove this log?

Reply:
This is duplicates the log line `Synced {} missing blocks from peer ...`, without the block refs.